lib/cocoapods/generator/copy_xcframework_script.rb

```diff
@@ -130,33 +130,6 @@ def script
   done
 }
 
-install_library() {
```

Comment:
@amorde I noticed this isn't called anywhere along with the one below.

are they used?

Reply:
these dont seem to be used, all tests pass and examples build now without it. I think its dead code